src/OpenBinary/reader.jl

```diff
@@ -2,10 +2,13 @@ Base.@kwdef mutable struct Reader <: PSRI.AbstractReader
     io::IOStream
 
     # stages
-    stage_total::Int
-    first_year::Int
-    first_stage::Int
     first_relative_stage::Int
+    first_stage::Int
+    last_stage::Int
+    num_stages::Int
+    stage_total::Int
```

Comment:
Do we need both num_stages and stage_total fields?

Reply:
It was important for normalizing the names while keeping track of their original meanings. I think we should discuss them before sticking to a new set of aliases and dropping the unnecessary ones.